package.json

```diff
@@ -138,7 +138,8 @@
   },
   "resolutions": {
     "fbjs/ua-parser-js": "^1.0.33",
-    "outdated-browser-rework/ua-parser-js": "^1.0.33"
+    "outdated-browser-rework/ua-parser-js": "^1.0.33",
+    "axios": "0.28.0"
```

Comment:
Was this dependency added when running `yarn install`?

Reply:
It wasn't. I added this manually. I ran `yarn why axios` and it seems like axios isn't a direct dependency, but a dependency on `@pact-foundation/pact@npm:10.2.2` and `wait-on@npm:6.0.1`.

Reply:
Ah I see. Then for transitive dependencies one can follow the [steps mentioned here](https://github.com/DataBiosphere/terra-ui/wiki/Security-and-Maintenance#dependabot-security-updates) to upgrade the dependency. I think the `package.json` can be updated directly only if the dependency is a direct dependency.

Reply:
Ah I see. Then for transitive dependencies one can follow the [steps mentioned here](https://github.com/DataBiosphere/terra-ui/wiki/Security-and-Maintenance#dependabot-security-updates) to upgrade the dependency. I think the package.json can be updated directly only if the dependency is a direct dependency.

Reply:
Note that this is the [`resolutions`](https://yarnpkg.com/configuration/manifest#resolutions) field, not `dependencies` / `devDependencies`. `resolutions` overrides the normal dependency resolution process for the listed dependencies.

Ideally, I think it would be nice to update dependencies higher in the tree (such as Pact in this case) to versions that pull in an updated version of the affected dependency (axios in this case). That way, we don't have to worry about `resolutions` producing conflicts or updating/removing entries from `resolutions` in the future. However, if there is no available version of Pact, etc. that results in axios being updated or if there's another reason why those intermediate dependencies cannot be updated, then `resolutions` is a useful tool to force the update.